### ISSUE TITLE
Комментарии к заявке

### ISSUE THREAD

**Issue by aakuptsov** [Justyful/balashov_ivr#1]
1. Название файла не соответствует правилам (см. образец)

2. Внутренняя вёрстка (оформление) заявки не соответствует правилам (см. образец)

3. Раздел "Похожие / аналогичные продукты": 

- в этом разделе не могут быть упомянуты бумажные справочники, поскольку они, очевидно, не являются программным продуктом и не могут анализироваться как аналоги; 

- Wolfram Alpha - это система вопросов / ответов, база знаний и набор алгоритмов, поэтому её тоже нельзя рассматривать как аналог, у Wolfram Alpha - слишком широкая область применения; 

Нужно произвести более детальный анализ аналогичных / похожих продуктов, потому что, есть подозрение, что они всё же есть.

4. Сам проект является по сути очень простым: это некая энциклопедия без интерактива (кроме поисковых запросов).

**Comment by aakuptsov** [comment]
>>Решение физических задач

Приложение будет само решать задачи?

**Comment by justyful** [comment]
Да, будет предложено ввести значения переменных и по алгоритмам задачка
должна будет решиться, иначе будет выведено сообщение о невозможности
(условно, как в photomath).

пт, 13 дек. 2019 г., 09:07 Alexander Kuptsov <notifications@github.com>:

**Comment by aakuptsov** [comment]
Зачёт

Комментарий от Салимова Д. Р.:  зачёт, но такому проекту нужен заказчик с ТЗ, например, кафедра естественных наук.
